docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,187 @@
+
+When installing the AvalancheGo client on the machines, unless you have a dedicated DevOps staff that will take care of node setup and configuration, we recommend using the [installer script](../nodes/build/set-up-node-with-installer.md) to set up the nodes. It will abstract most of the setup process for you, set up the node as a system service and will enable easy node upgrades.
+
+#### Cloud providers
+
+There are a number of different cloud providers. We have documents that show how to set up a node on the most popular ones:
+* [Amazon Web Services](../nodes/build/setting-up-an-avalanche-node-with-amazon-web-services-aws.md)
+* [Azure](../nodes/build/set-up-an-avalanche-node-with-microsoft-azure.md)
+* [Google Cloud Platform](../nodes/build/set-up-an-avalanche-node-with-google-cloud-platform.md)
+
+There is a whole range of other cloud providers that may offer lower prices or better deals for your particular needs, so it makes sense to shop around.
+
+Once you decide on a provider (or providers), if they offer instances in multiple data centers, it makes sense to spread the nodes geographically since that provides a better resilience and stability against outages.
+
+### Number of Validators
+
+Number of validators on a subnet is a crucial decision you need to make. For stability and decentralization, you should strive to have as many validators as possible.
+
+For stability reasons our recommendation is to have **at least** 5 full validators on your Subnet. If you have less than 5 validators your Subnet liveness will be at risk whenever a single validator goes offline, and if you have less than 4 even one offline node will halt your Subnet.
+
+Considering that at times you will have to take nodes offline, for routine maintenance (at least for node upgrades which happen with some regularity) or unscheduled outages and failures you need to be able to routinely handle at least one node being offline without your Subnet performance degrading.
+
+### Node bootstrap
+
+Once you set up the node instances and install AvalancheGo clients on them, nodes will need to boostrap (sync with the network). This is a lengthy process, as the nodes need to catch up and replay all the network activity since the genesis up to the present moment. Full bootstrap on a node can take more than a week, but there are ways to shorten that process, depending on your circumstances.
+
+#### State sync
+
+If the nodes you will be running as validators don't need to have the full transaction history, then you can use [state sync](../nodes/maintain/chain-config-flags.md#state-sync-enabled-boolean). With this flag enabled, instead of replaying the whole history to get to the current state, nodes simply download only the current state from other network peers, shortening the bootstrap process from multiple days to a couple of hours. If the nodes will be used for subnet validation exclusively, you can use the state sync without any issues.
```

Comment:
Note: we should specify state sync is just on the C-Chain

Reply:
Good call, will do.